src/components/cart/EmptyCart/EmptyCart.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Button from 'src/components/ui/Button'
+import { ShoppingCart as ShoppingCartIcon } from 'phosphor-react'
+import { useCartToggleButton } from 'src/sdk/cart/useCartToggleButton'
+
+function EmptyCart() {
+  const toggleProps = useCartToggleButton()
+
+  return (
+    <div data-testid="cart-empty-state" data-empty-cart>
+      <div data-empty-cart-title>
+        <ShoppingCartIcon size="32" />
+        <p>Your Cart is empty</p>
+      </div>
+      <Button {...toggleProps} variant="primary">
```

Comment:
I haven't checked, but what do those props do?

Reply:
It's to close the cart. It does the same thing as the `close` button in `CartSidebar.tsx`.